ios/FluentUI/Controls/IndeterminateProgressBarView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@available(*, deprecated, renamed: "IndeterminateProgressBarView")
+public typealias MSIndeterminateProgressBarView = IndeterminateProgressBarView
+
+/**
+Indeterminate progress bar view
+ */
+@objc(MSFIndeterminateProgressBarView)
+open class IndeterminateProgressBarView: UIView {
+	
+	/// progressbar's track
+	var trackLayer = CALayer()
+
+	/// gradient layer that slides across the track
+	var gradientLayer = CAGradientLayer()
+	
+	@objc override public init(frame: CGRect) {
+		super.init(frame: frame)
+	}
+	
+	required public init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	open override func layoutSubviews() {
+		super.layoutSubviews()
+		
+		trackLayer.frame = CGRect(x: 0, y: 0, width: frame.width, height: frame.height)
+		layer.addSublayer(trackLayer)
+	
+		addGradientLayer()
+		addAnimation()
+	}
+	
+	open override func didMoveToWindow() {
+		super.didMoveToWindow()
+		
+		if let window = window {
+			let lightColor : CGColor = Colors.primary(for: window).withAlphaComponent(0).cgColor
+			let darkColor : CGColor = Colors.primary(for: window).cgColor
+			
+			gradientLayer.colors = [lightColor, darkColor, lightColor]
+			trackLayer.backgroundColor = Colors.gray100.cgColor
+		}
+	}
+	
+	func addGradientLayer() {
+		gradientLayer.frame = CGRect(x: 0, y: 0, width: frame.width/2, height: frame.height)
+		gradientLayer.startPoint = CGPoint(x: 0.0, y: 1.0)
+		gradientLayer.endPoint = CGPoint(x: 1.0, y: 1.0)
+		gradientLayer.locations = [0.0, 0.5, 1.0]
+		layer.addSublayer(gradientLayer)
+	}
+
+	func addAnimation() {
+		let groupAnimation = CAAnimationGroup()
+		
+		let position = CABasicAnimation(keyPath: "position.x")
+		position.fromValue = -frame.width / 2
+		position.toValue = frame.width * 1.5
+		
+		let opacity = CAKeyframeAnimation(keyPath: "opacity")
+		opacity.keyTimes = [0, 0.375, 0.625, 1]
```

Comment:
>, 0.375, 0.625, 1 [](start = 23, length = 17)

all of these constant should be declared separately with static variables so we can modify easily later and know what each value means for reference